PLAN.md

```
@@ -0,0 +1,427 @@
+| `unrestricted` | Allow anyone to write | Removes `restricted` |
+| `open` | Accept join requests | Removes `closed` |
+| `visible` | Show metadata to anyone | Removes `hidden` |
+| `public` | Allow anyone to read | Removes `private` |
+| *(omit tag)* | Keep restriction active | `restricted` / `closed` / `hidden` / `private` present |
+
+Implementation must map between these two representations:
+- **Reading** kind 39000: presence of `restricted`/`private`/`hidden`/`closed` = flag is ON
+- **Writing** kind 9002: send `unrestricted`/`public`/`visible`/`open` to REMOVE a flag; omit to keep it
+
+#### Access control flags (in kind 39000 snapshot)
+| Flag | Meaning | Discord Equivalent |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)